content/projects/gsoc/2023/project-ideas/displaying_plugin_health_scores.adoc

```diff
@@ -0,0 +1,44 @@
+---
+layout: gsocprojectidea
+title: "Displaying of Plugin Health Scores"
+goal: "Display a possibly filtered view of Plugin Health Scores on plugins.jenkins.io and the details for each individual plugin."
+category: Tools
+year: 2023
+status: discussion
+sig: platform
+skills:
+- Java
+- YAML
+- Command line tools
+- Package management tool theory
+mentors:
+- "jleon"
+links:
+    emailThread: https://community.jenkins.io/t/gsoc2023-project-idea-displaying-of-plugin-health-scores/5318
+//   gitter: "jenkinsci/plugin-installation-manager-cli-tool"
+//   draft: https://docs.google.com/document/d/1s-dLUfU1OK-88bCj-GKaNuFfJQlQNLTWtacKkVMVmHc
+---
+// === Background
+// TBD
+//
+// === Quick Start
+// TBD
+//
+// === Skills to Study and Improve
+// * TBD
+//
+// === Project Difficulty Level
+// 
+// Beginner to Intermediate
+// 
+// === Project Size
+// 
+// 175 hours
```

Comment:
Hi @jmMeessen @jleon33 @kwhetstone @stopalopa @timja
Is this `175 hours` project size estimate based on a beginner student or an intermediate student?
Thank you.

Reply:
This has not been discussed yet. As you can see, the description is still in draft (commented out)
Please use https://community.jenkins.io/t/gsoc2023-project-idea-displaying-of-plugin-health-scores/5318 for the discussion (the project idea is still in discussion status)